src/scenes/mainScene.ts

```diff
@@ -314,6 +375,9 @@ export class MainScene extends Scene {
     }
 
     private drawEdges(cities: ICity[]) {
+        this.travelPathLines = this.add.graphics({
```

Comment:
this line occurs 3 times, consider extract method

Reply:
It's done. New pull request will follow immediately.